unit_tests/mocks/services/collection_service.py

```diff
@@ -38,7 +38,9 @@ def mock_update_collection(
             return create_collection_read_dto(import_id, data.title, data.description)
 
     # TODO: Think: Is this return value correct?
```

Comment:
Do we still need this TODO?

Reply:
I'll fix the todos in https://linear.app/climate-policy-radar/issue/PDCT-520/